### ISSUE TITLE
[xenial] W25 2018 Open store xenial version should be preinstalled with the image 

### ISSUE THREAD

**Issue by j2g2rp** [ubports/ubuntu-touch#694]
I upgraded from vivid to xenial RC week 25.
The installed version of the openstore wasn't working. I think that openstore compiled to xenial should be preinstalled. I don't know if this would be that easy or if this isn't a bug at all.

**Comment by j2g2rp** [comment]
yep, i had a lot of them. I upgraded from vivid to xenial without wiping. The openstore wasn't working. to close, to reboot but it wasn't working at all.
I'm going to ask for confirmation in spanish group.

If you need to know anything else don't hesiate in ask for it

**Comment by j2g2rp** [comment]
don't know if this can be usefulll. Some one was talking about this in the forum.
https://forums.ubports.com/topic/1370/attention-attention-big-news

**Comment by ruedigerkupper** [comment]
Confirming this problem. Upgraded from vivid to Xenial RC without wiping. Openstore not working. How can I install a working version of Openstore?

**Comment by ruedigerkupper** [comment]
Answering my own question (for anybody having the same problem):
1. Uninstall your installed version of openstore (not sure if strictly required, but I did so):
- go to apps scope
- long press openstore icon
- chose remove and confirm
2. Browse to open-store.io
- search for openstore
- download Xenial version
3. Open terminal (hoping you have it installed. If not, use adb or ssh to connect to your phone)
- cd Downloads
- pkcon install-local --allow-untrusted <name of click package you just downloaded>

**Comment by NeoTheThird** [comment]
If we made sure the version numbers for the xenial versions were higher, we would not have this problem. We could also just have the system settings app updater run once in the background on the first boot after an update/installation.

**Comment by UniversalSuperBox** [comment]
@NeoTheThird is 100% correct. The OpenStore in Xenial had a version bump today followed by a rebuild of the system image. All of the devices I've tested (Nexus 5, Fairphone 2, Nexus 7 flo) successfully used the preinstalled 2.41 rather than 2.40. It also removed version 2.40 from the user installation directory. 

Because it's now possible to upgrade all of your eligible apps after installing 16.04, I'll remove the critical label on this bug. I'll leave it up to @NeoTheThird to close it or not
